web/src/popover_menus.js

```diff
@@ -59,6 +92,7 @@ const default_popover_props = {
        its CSS to support Zulip's dark theme. */
     theme: "light-border",
     touch: true,
+    role: "popover",
```

Comment:
What does `role` do here? If it's the ARIA role, `"popover"` is not a valid value.

https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/tooltip_role

Reply:
Fixed, thanks for the catch!